web/templates/inbox.hbs

```diff
@@ -0,0 +1,10 @@
+<div id="inbox_main" class="no-select">
+    <div class="search_group" id="inbox_filters" role="group">
+        <i class="zulip-icon zulip-icon-search-inbox"></i>
+        <input type="text" id="{{INBOX_SEARCH_ID}}" value="{{search_val}}" autocomplete="off" placeholder="{{t 'Filter' }}" />
+        <button data-filter="include_muted" id="{{MUTED_FILTER_ID}}" type="button" class="btn btn-default btn-inbox-filter {{#if is_spectator}}fake_disabled_button{{/if}}" role="checkbox" aria-checked="false" tabindex="0">
+            <i id="inbox_filter_mute_toggle" class="fa fa-square-o"></i>
+            {{t 'Include muted' }}
+        </button>
+    </div>
+</div>
```

Comment:
Should this be called `inbox_view_header.hbs` or something? It seems to only have the search stuff at the top, but maybe I'm missing something since I don't see how the other templates are included yet.

Reply:
We append rest of the list here too. So, this is not just header.

Reply:
Hmm; maybe `inbox_view` or `inbox_main` would be a good name for this; and then perhaps we should have a comment  noting that detail? Usually we have partials or an empty div that the JS code inserts stuff into...

(Though cleaner might be to have an empty div for `inbox_view_body` in here so that the structure is explicit, if we're not doing a partial)

Reply:
Renamed and added that comment.